Document how to submit package to non default locations (#273)
yast/yast.github.io: doc/development.md

Manual Submission
-----------------

> ### Maintenance Branch

Reviewer: ```suggestion
### Maintenance Branches
```

> In maintenance branch automatic submissions are disabled and manual request is needed.

Reviewer: ```suggestion
In maintenance branches,the automatic submissions are disabled by default and a manual request is needed.
```

> It can be done by simple running of `rake osc:sr` in respective maintenance branch.

Reviewer: ```suggestion
It can be done by simply running of `rake osc:sr` in respective maintenance branch.
```

> It can be done by simply running of `rake osc:sr` in respective maintenance branch.

Reviewer: ```suggestion
It can be done by simply running `rake osc:sr` in the respective maintenance branch.
```

### Custom Build Service Repository

> There are two ways. The first one is usable if it needs more often submissions.

Reviewer: ```suggestion
There are two ways. The first one is useful if it needs more often submissions.
```
